crates/swift-integration-tests/src/string.rs

```diff
@@ -8,13 +8,19 @@ mod ffi {
 
     extern "Swift" {
         fn create_swift_string() -> String;
+        fn take_and_return_swift_string(arg: String) -> String;
```

Comment:
Let's call this `take_and_return_rust_string` and make the Swift function `(arg: RustString) -> RustString`

This way we have a test for both `-> String` and `-> RustString`